install/froxlor.sql

```diff
@@ -684,8 +685,12 @@ opcache.interned_strings_buffer'),
 	('panel', 'password_special_char_required', '0'),
 	('panel', 'password_special_char', '!?<>§$%+#=@'),
 	('panel', 'customer_hide_options', ''),
-	('panel', 'version', '0.9.39.5'),
-	('panel', 'db_version', '201809280');
+	('system', 'apache_use_nrp', '0'),
+	('system', 'proxyconf_vhost', '/etc/nginx/sites-enabled/')
+        ('system', 'default_proxyconf', '')
+	('system', 'proxyreload_command', 'service nginx reload')
+	('panel', 'version', '0.9.39.6-dev1'),
```

Comment:
please do not increase or change version field...only increase db_version (date + n) - also, it is not necessary to append -devX anywhere

Reply:
I was not sure how to handle the versioning correctly for branched versions. I hope I fixed it accordingly.